api/nodes/views.py

```diff
@@ -665,15 +666,15 @@ class NodeChildrenList(JSONAPIBaseView, bulk_views.ListBulkCreateJSONAPIView, No
     ordering = ('-modified',)
 
     def get_default_queryset(self):
-        return default_node_list_queryset(model_cls=Node)
+        auth = get_user_auth(self.request)
+        return default_node_list_permission_queryset(auth=auth, model_cls=Node)
```

Comment:
Using `default_node_list_permission_queryset` to match RegistrationChildrenList.  (Not making RegistrationChildrenList subclass NodeChildrenList though, because RegistrationChildrenList has limited functionality.